interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -205,6 +206,18 @@ namespace cling {
 
     std::string errMsg;
     DyLibHandle dyLibHandle = platform::DLOpen(canonicalLoadedLib, &errMsg);
+    llvm::StringRef EM(errMsg);
+    // If error message contains "undefined symbol", it means that we failed to load dependency
+    // libraries. We emit callback to LazyFunctionCallback with undefined symbol and try to load
+    // the library which has undef's definition.
+    if (EM.contains("undefined symbol: ")) {
```

Comment:
Can we get multiple 'undefined symbol' errors?

Reply:
Yes, by calling `return loadLibrary(libStem, permanent, resolved)` recursively